packages/rollup/README.md

```diff
@@ -100,7 +100,7 @@ ___
 
 Which paths to include when processing images.
 
-**`default`** '**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}?*'
+**`default`** ['**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}', '**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}?*']
```

Comment:
This renders weirdly in markdown, maybe you need to somehow escape the asterisks?

Reply:
fixed. escaping the asterisks worked, but I decided to put it in backticks instead, which also worked and looked a bit better to me